### PR TITLE
477 release gsdesign2 v113

### PR DESCRIPTION
**Pull request by LittleBeannie**
This PR will be merged after the PR #479  is merged.

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+## User interface Improvements
+
+- Enable passing named vector of `col_decimals` and `analysis_decimals` to `summary.gs_design()` (#403, #431, @jdblischak).
+
+## Coding practive Improvements
+
+- Add robust check if `lower` is equivalent to `gs_b` (#413, thanks to @jdblischak )
+- The `summary()`, `as_gt()` `as_rtf()`, and `to_integer()` functions are refactored (#448, #449, #450, #465, #461, thanks to @yihui).
+- Remove the `full_alpha` argument from `as_rtf.gs_design()` (#458, thanks to @yihui).
+- Other coding practice improvement wroth mentioning (#409, #412, #429, #437, #440, #447, #453, #464, #467, $475, thanks to @yihui, @nanxstats, and @jdblischak).
```
Comment (jdblischak) [inline NEWS.md:32]:
```suggestion
- Other coding practice improvement worth mentioning (#409, #412, #429, #437, #440, #447, #453, #464, #467, $475, thanks to @yihui, @nanxstats, and @jdblischak).
```

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+
+## Documentation
+
+- Update the documentation of `gs_b()` (#415, @jdblischak)
+
+## Testing
+
+- More Developer tests of `gs_power_ahr()` are added (#420, @LittleBeannie).
+- More Developer tests of `summery()` is added (#422, #426, thanks to @yuliasidi, @jdblischak and @LittleBeannie).
```
Comment (jdblischak) [inline NEWS.md:41]:
```suggestion
- More Developer tests of `summary()` is added (#422, #426, thanks to @yuliasidi, @jdblischak and @LittleBeannie).
```

**Comment by nanxstats** [comment]
With this many [bug fixes, new features, and changes that are backward compatible](https://r-pkgs.org/lifecycle.html#sec-lifecycle-release-type), 1.2.0 might be more reasonable.

File: NEWS.md (hunk 2 above)
Comment (LittleBeannie) [reply NEWS.md]: Thank you so much for pointing out the 2 typos. They are corrected in the latest commit.

**Review by jdblischak — APPROVED** [review]
I'm fine with either 1.1.3 or 1.2.0

**Comment by jdblischak** [comment]
Just remembered that we need to update `cran-comments.md` to mention that we are aware of the following `NOTE`, and that we also maintain {simtrial}:

```
* checking dependencies in R code ... NOTE
Missing object imported by a ':::' call: ‘simtrial:::as_gt.simtrial_gs_wlr’
```

**Comment by LittleBeannie** [comment]
> I'm fine with either 1.1.3 or 1.2.0

Thanks for proposing the alternative. I will confirm with Keaven to check if he has any preference.

**Pull request merged**